compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Conversions.scala

```diff
@@ -251,12 +251,7 @@ final class Conversions(
         builder.setScenarioPartyAlreadyExists(party)
 
       case Error.UserManagement(err) =>
```

Comment:
Do we need this error case in the Scala AST? It seems like we should never hit this now?

Reply:
I removed it.

Reply:
In fact we now don't need a type for `UserManagementError` in daml-script at all... It remained used only to communicate from `UserManagementStore` to `IdeLedgerClient`, where it was reduced anyway to just `Option`.

I changed to have the same `Option` based interface also in `UserManagementStore`.

And code just gets removed!